Command/DebugCommand.php

```diff
@@ -0,0 +1,172 @@
+<?php
+
+declare(strict_types=1);
+
+namespace SofaScore\Purgatory\Command;
+
+use SofaScore\Purgatory\Mapping\Loader\LoaderInterface;
+use SofaScore\Purgatory\Mapping\MappingCollection;
+use SofaScore\Purgatory\Mapping\MappingValue;
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Input\InputArgument;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+use Symfony\Component\Routing\RouteCollection;
+use Symfony\Component\Routing\RouterInterface;
+
+
+class DebugCommand extends Command
+{
+    private const ARGUMENT_FILTER = 'filter';
+
+    protected static $defaultName = 'purgatory:debug';
+
+    private LoaderInterface $loader;
+    private RouterInterface $router;
+    private ?RouteCollection $routeCollection = null;
+
+    public function __construct(LoaderInterface $loader, RouterInterface $router)
+    {
+        $this->loader = $loader;
+        $this->router = $router;
+        parent::__construct();
+    }
+
+    protected function configure(): void
+    {
+        parent::configure();
+
+        $this->setDescription('Dumps route and purgatory information for changed entity given as argument');
+
+        $this->addArgument(
+            'filter',
+            InputArgument::OPTIONAL,
+            "Entity class name with optional property name and sub-properties (e.g. Event, Event::startDate, Event::homeScore.period1)"
+        );
+    }
+
+    public function execute(InputInterface $input, OutputInterface $output): int
+    {
+        $this->dump($this->loader->load(), $output, $input->getArgument(self::ARGUMENT_FILTER));
+        return 0;
+    }
+
+    private function dump(MappingCollection $mappings, OutputInterface $output, ?string $filter): void
+    {
+        foreach ($mappings as $entityOrProperty => $mappingValues) {
+            if (null !== $filter && !$this->itemMatchesFilter($entityOrProperty, $filter)) {
+                continue;
+            }
+
+            $output->writeln($entityOrProperty);
+            foreach ($mappingValues as $mapping) {
+                $this->dumpMappingValue($mapping, $output, 1);
+                $output->writeln('');
+            }
+        }
+    }
+
+    private function dumpMappingValue(MappingValue $mappingValue, OutputInterface $output, int $indent = 0): void
+    {
+        $this->dumpMappingValueStringElement(null, $mappingValue->getRouteName(), $output, $indent);
+        $this->dumpMappingValueStringElement('path', $this->describePath($mappingValue), $output, $indent + 1);
+        $this->dumpMappingValueArrayElement('parameters', $mappingValue->getParameters(), $output, $indent + 1);
+        $this->dumpMappingValueStringElement('priority', $mappingValue->getPriority(), $output, $indent + 1);
+        $this->dumpMappingValueStringElement('if', $mappingValue->getIf(), $output, $indent + 1);
+        $this->dumpMappingValueArrayElement('tags', $mappingValue->getTags(), $output, $indent + 1);
```

Comment:
priority and tags are not used anywhere and WebCacheInterface doesn't support any meta for url purging so I would remove that altogether from the project for the first public version. If the need arises we can make a new advanced interface that would receive meta data.